/camera_topic: "/camera/color/image_raw"
/network_height: 256
/network_width: 256